app/src/main/java/com/neosensory/whatsup/Satellite.java

```diff
@@ -282,12 +273,11 @@ public String getName() {
    */
   public String[] getTles() {
     if (hasTle) {
-      String[] tles = {tle1, tle2};
-      return tles;
+      String[] tles = {tle1, tle2}; 
     } else {
       String[] tles = {"", ""};
-      return tles;
     }
+    return tles;
```

Comment:
Hi @sydneysauer: this modification does not work as the variable `tles` is not defined outside of the if/else structure. A more appropriate modification might be` String[] tles = {"",""} if(hasTle){...` and then get rid of the `else` statement.

Reply:
Thanks for catching that...my Java professor would be ashamed of me. :) In that case, the original implementation was simpler than creating one return. I reverted the changes and pushed so it should be OK now.

Reply:
@GreatScott